### ISSUE TITLE
Homematic Hand Transmitter HM-RC-4-3 Values remain "True" after single usage

### ISSUE THREAD

**Issue by cherberg** [ioBroker/ioBroker.hm-rpc#1113]
**Describe the bug**
I have added a HM-RC-4-3 to my Homematic CCU instance. Pairing went fine and the iobroker adapter is showing the device properly. There are four channels for the four buttons of the device, each showing 5 Datapoints (see screenshot). The Numbering does not relfect the physical buttons (Top Button on the remote is Channel 2 in the adapter) but this is more cosmetic and not a big issue.

My problem is the following:
Initially all values for the 5 Datapoints were showing false, except the first one INSTALL_TEST. Pushing the buttons correctly changes the value from False to True but only once as it is not switching back and remains "True". After that the same  button action is no longer recorded by the adapter. I can manually set the value back to False and then the Button action again changes it to True but then again it remains in this state. I cannot use the remote to trigger actions via script.

**To Reproduce**
1. Add / Pair an HM-RC-4-3 to a CCU2
2. Check the RPC Adapter and navigate to the new device entry
3. Push a remote button Short or Long and see the respective value changing from False to True
4. See that this state remains and does not switch back to False

**Expected behavior**
I would expect the Value to change from False to True but then return to False and remain in this state

**Screenshots & Logfiles**
See that all values are static "True". Button actions does not trigger any change
![image](https://github.com/ioBroker/ioBroker.hm-rpc/assets/8701752/52bf58b1-0859-4a73-9f56-de18286bd648)

**Versions:**
 - hm-rpc version: 1.15.19 
 - hm-rega version: 3.0.47
 - JS-Controller version: 5.0.16
 - Node version: v18.16.0
 - Operating system: Raspbian
 - CCU model: CCU2
 - CCU firmware: 2.61.7

**Additional context**
Appreciate your help in this matter.

**Comment by Apollon77** [comment]
Yes this is "as designed by Homematic" and it that way for all buttons. For button presses please react on "update" and not on "change" because they are set tto true again when pressed again.
